spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -231,8 +234,8 @@ def reset(self):
         # Reset the policies and environments via the controllers
         raise NotImplementedError
 
-    @multimethod
-    def nav(self, nav_target: str = None) -> Tuple[bool, str]:
+    @multimethod  # type: ignore
+    def nav(self, nav_target: str = None) -> Tuple[bool, str]:  # type: ignore
```

Comment:
can we remove this #type: ignore stuff?

Reply:
Consulted with @jimmytyyang for passing pre-commit checks, and was asked to keep it since pre-commit checks were failing.